apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.Tests/Trace/GrpcTraceConsumerTest.cs

```diff
@@ -39,7 +40,7 @@ public void Receive()
             Task<TraceServiceClient> taskClient = Task.FromResult(mockClient.Object);
             GrpcTraceConsumer consumer = new GrpcTraceConsumer(taskClient);
 
-            consumer.Receive(traces);
+            consumer.Receive(new[] { traces.Traces_[0] });
```

Comment:
Perhaps just `traces.Traces_` or `traces.Traces_.ToArray()`? We wouldn't want to actually only receive the first trace, would we?